services/src/test/java/io/scalecube/services/TransportSenderTest.java

```diff
@@ -0,0 +1,65 @@
+package io.scalecube.services;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertNotEquals;
+
+import io.scalecube.transport.Transport;
+
+import org.junit.Test;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+public class TransportSenderTest {
+
+  @Test
+  public void test_transport_sender() throws InterruptedException {
+
+    Transport transport = Transport.bindAwait();
+    TransportSender sender = new TransportSender(transport);
+
+    assertEquals(transport.address(), sender.address());
+
+    CountDownLatch latch = new CountDownLatch(1);
+    sender.listen().subscribe(onNext -> {
+      latch.countDown();
+    });
+
+    latch.await(1, TimeUnit.SECONDS);
```

Comment:
all what this line of code is doing is just waiting for 1 second... since you are not sending any message